cdc/log.cc

```diff
@@ -989,24 +992,34 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
                 // Note: further improvement here would be to coalesce the pre-image selects into one
                 // iff a batch contains several modifications to the same table. Otoh, batch is rare(?)
                 // so this is premature.
+                tracing::trace(tr_state, "CDC: Selecting preimage [mutation #{}]", idx);
```

Comment:
There should be a message in the other branch as well. A missing message is strictly worse than a negative message, there is always a chance that a message is missing due to a bug and this creates uncertainty.

Reply:
Right. To sum it up: please, make sure to have at least one tracing message on each branch tree that you decided to trace.

Reply:
Added an `else` branch with an appropriate message.